packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfUserControlLibrary-VisualBasic/UserControl1.xaml

```diff
@@ -0,0 +1,12 @@
+<UserControl x:Class="Company.WpfControlLibrary.UserControl1"

```

Comment:
The .NET Framework VB Template looks like this: 

```XAML
<UserControl x:Class="UserControl1"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
             xmlns:local="clr-namespace:usercontrol"
             mc:Ignorable="d" 
             d:DesignHeight="450" d:DesignWidth="800">
    <Grid>
            
    </Grid>
</UserControl>
```
Note the absence of `Company.WpfControlLibrary` prefix. Since tthe VB templates do not wrap themselves inside of a namespace, I don't think it should get a `Company.WpfControlLibrary` prefix here either.